config/secrets.yml

```diff
@@ -25,6 +25,8 @@ development:
   mailer_address: <%= ENV["MAILER_ADDRESS"] %>
   mailer_port: <%= ENV["MAILER_PORT"] %>
   mailer_authentication: <%= ENV["MAILER_AUTHENTICATION"] %>
+  google_client_id: <%= ENV['GOOGLE_CLIENT_ID'] %>
```

Comment:
Don't forget about production and the testing environment :P

Reply:
noooo again lol